pkg/cgroup/stat_reader_test.go

```diff
@@ -0,0 +1,147 @@
+		imap["memory.current"] = 100
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_memory_usage_bytes"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(100))
+	})
+	It("Test converter cgroupfs_memory_usage_bytes with memory.usage_in_bytes", func() {
+		imap := make(map[string]interface{})
+		imap["memory.usage_in_bytes"] = 100
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_memory_usage_bytes"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(100))
+	})
+	It("Test converter cgroupfs_kernel_memory_usage_bytes", func() {
+		imap := make(map[string]interface{})
+		imap["memory.kmem.usage_in_bytes"] = 100
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_kernel_memory_usage_bytes"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(100))
+	})
+	It("Test converter cgroupfs_tcp_memory_usage_bytes", func() {
+		imap := make(map[string]interface{})
+		imap["memory.kmem.tcp.usage_in_bytes"] = 100
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_tcp_memory_usage_bytes"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(100))
+	})
+	It("Test converter cgroupfs_cpu_usage_us with cpuacct.usage", func() {
+		imap := make(map[string]interface{})
+		imap["cpuacct.usage"] = uint64(100000)
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_cpu_usage_us"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(uint64(100)))
+	})
+	It("Test converter cgroupfs_cpu_usage_us with usage_usec", func() {
+		imap := make(map[string]interface{})
+		imap["usage_usec"] = 100
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_cpu_usage_us"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(100))
+	})
+	It("Test converter cgroupfs_system_cpu_usage_us with cpuacct.usage_sys", func() {
+		imap := make(map[string]interface{})
+		imap["cpuacct.usage_sys"] = uint64(100000)
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_system_cpu_usage_us"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(uint64(100)))
+	})
+	It("Test converter cgroupfs_system_cpu_usage_us with system_usec", func() {
+		imap := make(map[string]interface{})
+		imap["system_usec"] = 100
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_system_cpu_usage_us"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(100))
+	})
+	It("Test converter cgroupfs_user_cpu_usage_us with cpuacct.usage_user", func() {
+		imap := make(map[string]interface{})
+		imap["cpuacct.usage_user"] = uint64(100000)
+		out := convertToStandard(imap)
+		Expect(len(out)).To(Equal(1))
+		v, ok := out["cgroupfs_user_cpu_usage_us"]
+		Expect(ok).To(Equal(true))
+		Expect(v).To(Equal(uint64(100)))
+	})
+	It("Test converter cgroupfs_system_cpu_usage_us with user_usec", func() {
```

Comment:
could you also change `cgroupfs_system_cpu_usage_us` to `cgroupfs_user_cpu_usage_us `?

The other updates look good to me.